source/engine/graphics/api/vulkan/Initializers.cpp

```diff
@@ -40,7 +40,7 @@ auto CreatePipelineShaderStageCreateInfo(ShaderStage stage, const vk::ShaderModu
     };
 }
 
-vk::PipelineVertexInputStateCreateInfo CreateVertexInputCreateInfo(const vk::VertexInputBindingDescription& vertexInputDesc, const std::array<vk::VertexInputAttributeDescription, 5>& vertexAttributeDesc)
+vk::PipelineVertexInputStateCreateInfo CreateVertexInputCreateInfo(const vk::VertexInputBindingDescription& vertexInputDesc, const std::array<vk::VertexInputAttributeDescription, 7>& vertexAttributeDesc)
```

Comment:
Not major, but this could take a span instead so it doesn't need to be touched if the size changes again.